src/hotspot/share/nmt/memReporter.cpp

```diff
@@ -524,7 +535,7 @@ void MemSummaryDiffReporter::report_diff() {
     MemTag mem_tag = NMTUtil::index_to_tag(index);
     // thread stack is reported as part of thread category
     if (mem_tag == mtThreadStack) continue;
-    diff_summary_of_type(mem_tag,
+    diff_summary_of_tag(mem_tag,
```

Comment:
Move out into mainline PR

Reply:
Fixed in commit "missed flag/tag -> type" (7640108).